src/sfError.ts

```diff
@@ -74,6 +88,10 @@ export class SfError<T = unknown> extends NamedError {
     }
   }
 
+  public get fullStack(): string | undefined {
```

Comment:
this existed on NamedError.  Now it recurses through the `cause` prop's error chain

Reply:
https://github.com/forcedotcom/kit/blob/963b1d1f34833fbd889a89f18ed82194aeb7f5a5/src/errors.ts#L34

Reply:
this method was only used (at least in `salesforcecli`) by sfdxCommand:
https://github.com/search?q=org%3Asalesforcecli%20%2F%5C.fullStack%2F&type=code

we could remove it and recommend to use `util.inspect(err)`, it checks `error.cause` and pretty-print the stacktrace while also removing duplicates:
https://nodejs.org/dist/latest-v20.x/docs/api/errors.html#errorcause

```js
const util = require('node:util')
const badServer = new Error('Bad server')
const cause = new Error('The remote HTTP server responded with a 500 status', { cause: badServer });
const symptom = new Error('The message failed to send', { cause });

console.log(util.inspect(symptom));
```
```
Error: The message failed to send
    at Object.<anonymous> (/Users/cdominguez/code/gh/sf/command/index.js:4:17)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    ... 4 lines matching cause stack trace ...
    at node:internal/main/run_main_module:28:49 {
  [cause]: Error: The remote HTTP server responded with a 500 status
      at Object.<anonymous> (/Users/cdominguez/code/gh/sf/command/index.js:3:15)
      at Module._compile (node:internal/modules/cjs/loader:1376:14)
      ... 4 lines matching cause stack trace ...
      at node:internal/main/run_main_module:28:49 {
    [cause]: Error: Bad server
        at Object.<anonymous> (/Users/cdominguez/code/gh/sf/command/index.js:2:19)
        at Module._compile (node:internal/modules/cjs/loader:1376:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
        at Module.load (node:internal/modules/cjs/loader:1207:32)
        at Module._load (node:internal/modules/cjs/loader:1023:12)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
        at node:internal/main/run_main_module:28:49
  }
}
```

Reply:
yeah, I think I like that.  I did a broader github search and couldn't find anything else using it, either.

I didn't know about `util.inspect`... that's cool